template/src/app-state/useVirtualBackground.native.ts

```diff
@@ -0,0 +1,167 @@
+import {useEffect, useRef} from 'react';
+import {
+  Option,
+  useVB,
```

Comment:
useVirtualBackground

Reply:
his is for core , have updated methods in customization vb api